### PR TITLE
Cockpit 282

### PR DESCRIPTION
**Pull request by garrett**
Setting this as draft for now, as it's in preview mode.

Preview:
https://garrett.github.io/cockpit-project.github.io/blog/cockpit-282.html

### CODE REVIEW

**Comment by marusak** [comment]
Let's also mention RTL basic support

**Comment by garrett** [comment]
We still need to update the links and add a summary and such, but I'm marking this ready for review already.

File: _posts/2022-12-14-cockpit-282.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Right-to-left language support
+
+Right-to-left language support has been enabled for languages that require it. Our most complete RTL language is Hebrew. When choosing Hebrew in the languages list, the layout will shift and become right-aligned.
+
+In addition to basic RTL support, we have started to modify Cockpit and PatternFly to work with RTL languages in a better way. This is not complete, but we wanted to enable this as soon as possible, so people could use it and report bugs.
+
+We are aware that there are some basic issues across Cockpit. Most of Cockpit should be usable in RTL, but there are still some alignment and spacing problems.
+
+The most affected pages at the moment are:
+- the login page (which does not currently adapt to RTL)
```
Comment (marusak) [inline _posts/2022-12-14-cockpit-282.md:34]: Not true. It works just fine (but you need to mount the static dir locally).

File: _posts/2022-12-14-cockpit-282.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- optional pages such as `cockpit-podman`, `cockpit-machines`, and `cockpit-ostree`
+- all 3rd party app pages
+
+If you know Hebrew, then please help us with the implementation. If you know other RTL languages, we're happy to work with you to help improve translations.
+
+We will try to further improve RTL to the best of our ability, but need help from people who actually know Hebrew to make sure it is correct.
```
Comment (garrett) [inline _posts/2022-12-14-cockpit-282.md:40]:
```suggestion
Right-to-left language support has been enabled for languages that require it. Our most complete RTL language is Hebrew. When choosing Hebrew in the languages list, the layout will shift and become right-aligned.

In addition to basic RTL support, we have started to modify Cockpit and PatternFly to work with RTL languages in a better way. This is not complete, but we wanted to enable this as soon as possible, so people could use it and report bugs.

We are aware that there are some basic issues across Cockpit. Most of Cockpit should be usable in RTL, but there are still some alignment and spacing problems.

We will try to further improve RTL to the best of our ability, but need help from people who actually know Hebrew to make sure it is correct. If you know Hebrew, then please help us with the implementation. If you know other RTL languages, we're happy to work with you to help improve translations.
```

Comment (garrett) [reply _posts/2022-12-14-cockpit-282.md:40]: An even shorter version:

```suggestion
Right-to-left language support has been enabled for languages that require it. When selecting a RTL language from the list, the layout will shift to be right-aligned.

We have also started modifying Cockpit and PatternFly to work better with RTL languages. However, there are still some alignment and spacing issues. We will continue to work to fix the more obvious problems.

If you know Hebrew (Cockpit's most complete RTL translation) or other RTL languages, please help us improve the implementation and translations by [filing issues on GitHub](https://github.com/cockpit-project/cockpit/issues/new?labels=bug,i18n), sending pull requests, and/or [improving translations on weblate](https://translate.fedoraproject.org/projects/cockpit/main/).
```

File: _posts/2022-12-14-cockpit-282.md (hunk 1 above)
Comment (garrett) [reply _posts/2022-12-14-cockpit-282.md]: Whoops! I forgot about the login screen being a special case. Thanks! Changed the intro based on this comment and our F2F discussion about it.

**Review by KKoukiou — COMMENTED** [review]
THe bodhi links are not the exact new links but just the link to the page. If you want to replace these are available already.

**Review by KKoukiou — APPROVED** [review]

**Comment by garrett** [comment]
Yeah, there's no way for the script to know what the correct links are, so it fills in placeholders that get to the right place generally.

 I'll follow those links and update it with the new ones... and make sure we have a summary too.

File: _posts/2022-12-14-cockpit-282.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 282
+author: garrett
+date: '2022-12-14'
+tags: cockpit
+slug: cockpit-282
+category: release
+summary: ''
```
Comment (garrett) [inline _posts/2022-12-14-cockpit-282.md:8]:
```suggestion
summary: 'Accounts page redesign and right-to-left language support'
```

File: _posts/2022-12-14-cockpit-282.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+## Try it out
+
+Cockpit 282 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/282)
+* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
```
Comment (garrett) [inline _posts/2022-12-14-cockpit-282.md:43]:
```suggestion
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2022-14210d1ef5)
* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-9eabd9cf61)
```

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**